frontend/javascripts/test/sagas/volumetracing/volumetracing_saga_integration.spec.js

```diff
@@ -506,13 +536,130 @@ test.serial("Brushing/Tracing with undo (I)", async t => {
   Store.dispatch(addToLayerAction(paintCenter));
   Store.dispatch(finishEditingAction());
 
+  if (assertBeforeUndo) {
```

Comment:
Could you please add a comment similar to line 528 and 533 as you did it in line 594?
> `// Brush with ${newCellId}`

This makes the test easier / faster to understand.